device/ethernet.yaml

```diff
@@ -47,46 +59,46 @@ components:
           x-constraint:
           - '/components/schemas/Port/properties/name'
           - '/components/schemas/Lag/properties/name'
-          x-field-uid: 1
+          x-field-uid: 2
```

Comment:
Please fix x-field-uid s such that previously existing ones dont change. Probably needs a bug logged against otg tools since it does not seem to be detecting this issue which causes backward compatibility break in .proto . I believe tools should probably be taking care of this mistake at compile time ( build.py ) and ci pipeline if submitter makes this mistake.

Reply:
ok and agreed.